### ISSUE TITLE
mkmozconfig gives me `ImportError: No module named mozconfigwrapper` and fails to create the file

### ISSUE THREAD

**Issue by dholbert** [ahal/mozconfigwrapper#10]
Filing issue on this set of errors, which I've been hitting whenever I try to use mkmozconfig:

```
$ mkmozconfig test
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named mozconfigwrapper
Created: /home/dholbert/.mozconfigs/test
Error: /home/dholbert/.mozconfigs/test does not exist
```

Both the `ImportError` and the `Error` seem unexpected.

I'm just trying out `mozconfigwrapper` for the first time - I installed it via `pip`, and I added the `mozconfigwrapper.sh` file to my bashrc, and I created the `/home/dholbert/.mozconfigs/` directory just for good measure.  I'm not sure if there's another step I might be missing...

Other commands do work, e.g. `buildwith`, `mozconfig -l`, `mozconfig -e` (after I've manually added a file with the appropriate name in `/home/dholbert/.mozconfigs/`).  It's specifically `mkmozconfig` that is erroring out.

I'm using Ubuntu 20.10 on a Dell XPS laptop.

**Comment by dholbert** [comment]
(Note: the output does say `Created:  /home/dholbert/.mozconfigs/test` but in fact the file does not get created.)

**Comment by dholbert** [comment]
Gijs says he hit this as well, so it's not just me (yay!)

**Comment by dholbert** [comment]
(@ahal - hoping you can take a look, if you have cycles. Let me know if you need any other info here; happy to try diagnostics/etc. if you happen to be unable to repro locally.)

**Comment by whimboo** [comment]
@ahal is currently not available. So you will have to wait or we can find someone else to investigate / fix it.

**Comment by dholbert** [comment]
Ah, my mistake - I thought he'd just returned from some time away (saw some back & forth on Matrix about being back, & thought I'd bump this issue to be sure it didn't get lost & since I suspect it's something trivial/silly).

No huge rush on this, though.

**Comment by ahal** [comment]
I did see this while in leave but forgot about it, so thanks for the bump! I've added a task to investigate. I do suspect it'll be relatively simple to fix.

**Comment by ahal** [comment]
I wasn't able to reproduce in my Linux environment, but could with mozilla-build. This seems to happen because the `mkmozconfig` shell function calls `python`, so if you install `mozconfigwrapper` into a different Python installation than the one pointed to by `python`, you'll hit this error.

Changing the `python` call to `python3` fixed the issue for me (as with `mozilla-build` python still refers to Python 2). Though if I make this change, the bug will still exist in the opposite case (if someone has it installed in their Python 2 environment. I guess I should just drop Python 2 support and change it to Python 3.

**Comment by ahal** [comment]
This should be fixed in mozconfigwrapper>=1.0.0 which is now published. Though the core of the issue still exists if `python3` doesn't refer to the installation where `mozconfigwrapper` was installed.

**Comment by dholbert** [comment]
After running `pip3 install --user -U mozconfigwrapper` to upgrade, and then opening a new shell, I can't reproduce this anymore. Thanks!
